projects/angular-calendar/src/modules/month/calendar-month-row.component.ts

```diff
@@ -0,0 +1,205 @@
+import {
+  Component,
+  Input,
+  OnChanges,
+  SimpleChanges,
+  TemplateRef,
+} from '@angular/core';
+import { CalendarEvent, MonthView, WeekDay } from 'calendar-utils';
+
+@Component({
+  selector: 'mwl-calendar-month-row',
+  template: `
+    <div class="cell-day" *ngFor="let day of daysSliced">
+      <div *ngFor="let note of notePerDay.get(day)">
+        <ng-template
+          [ngTemplateOutlet]="cellMonthNoteTemplate"
+          [ngTemplateOutletContext]="{
+            day: day,
+            firstDayOfRow: firstDayOfRow,
+            note: note
+          }"
+        >
+        </ng-template>
+      </div>
+    </div>
+  `,
+  styleUrls: ['./calendar-month-row.scss'],
+})
+export class CalendarMonthRowComponent implements OnChanges {
+  @Input() notes: CalendarEvent[];
+  @Input() view: MonthView;
+  @Input() rowIndex: number;
+  @Input() cellMonthNoteTemplate: TemplateRef<any>;
+
+  startDate: Date;
+  endDate: Date;
+
+  daysSliced: WeekDay[] = [];
+  firstDayOfRow: WeekDay;
+
+  concernedNotes: CalendarEvent[];
```

Comment:
i'm not a big fan on this naming, i'm not sure what you want to say with "concerned" ?

Reply:
notesOfCurrentRow ?

Reply:
i'd prefer `currentRowNotes` (i'm not a big fan of using 'of' in variables name)